src/gamekit.cs

```diff
@@ -569,6 +698,13 @@ interface GKScore : NSSecureCoding {
 		[Async]
 		void ReportScores (GKScore[] scores, [NullAllowed] GKChallenge[] challenges, [NullAllowed] Action<NSError> completionHandler);
 
+		[NoWatch]
+		[iOS (14,0)][Mac (10,16)][TV (14,0)]
+		[Static]
+		[Async]
+		[Export ("reportLeaderboardScores:withEligibleChallenges:withCompletionHandler:")]
+		void ReportLeaderboardScores (GKLeaderboardScore[] scores, GKChallenge[] challenges, [NullAllowed] Action<NSError> completionHandler);
```

Comment:
Very minor, but this sounds a bit more descriptive:

```suggestion
		void ReportLeaderboardScores (GKLeaderboardScore[] scores, GKChallenge[] eligibleChallenges, [NullAllowed] Action<NSError> completionHandler);
```